src/ecrc-frontend/src/components/page/orgverification/OrgVerification.js

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import "../page.css";
+import Header from "../../base/header/Header";
+import Footer from "../../base/footer/Footer";
+import SideCard from "../../base/sideCard/SideCard";
+import { Button } from "../../base/button/Button";
+import Table from "../../composite/table/Table";
+
+export default function OrgVerification({
+  page: {
+    pageLayout: { header, sideCard1, sideCard2 },
+    org,
+    setOrg
+  }
+}) {
+  useEffect(() => {
+    window.scrollTo(0, 0);
```

Comment:
hmm what is this for? is this scrolling to 0 horizontally and 0 vertically, isnt that default behaviour?

Reply:
It is scrolling to top left of the page, it is default on sites with a page load, but because react is just rendering components, when it had the old text (which was long) when it rendered the org verification page, it stayed at the same scroll level...which on small window sizes, was the bottom. Not sure if/how itll react to routes...or if itll be needed in the end, but when entering an org code and hitting enter, it would just show the bottom of the org ver page on mobile...

Reply:
ok let's revisit this later and see if it still required, thanks